components/eventing-controller/pkg/env/nats_config.go

```diff
@@ -25,8 +25,14 @@ type NatsConfig struct {
 	IdleConnTimeout     time.Duration `envconfig:"IDLE_CONN_TIMEOUT" default:"10s"`
 
 	// JetStream-specific configs
-	JSStreamName            string `envconfig:"JS_STREAM_NAME" default:"kyma"`
-	JSStreamStorageType     string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Name of the JetStream stream where all events are stored.
+	JSStreamName string `envconfig:"JS_STREAM_NAME" default:"kyma"`
+	// Storage type of the stream, memory or file.
+	JSStreamStorageType string `envconfig:"JS_STREAM_STORAGE_TYPE" default:"memory"`
+	// Retention policy specifies when should events be deleted from the stream.
+	//  interest: when all known observables have acknowledged a message it can be removed.
```

Comment:
out-of-scope: Is there a rule when to make empty spaces before comments in golang ? 
I know it is used to create code blocks, e.g. [src](https://github.com/natefinch/godocgo/blob/eb5b06972700b1bb45d8b9a8316cfc11b1faadc7/sub/doc.go#L63) => [rendered](https://pkg.go.dev/github.com/natefinch/godocgo@v0.0.0-20160109030932-eb5b06972700/sub#hdr-What_gets_displayed)

Reply:
Not sure. In my case, it is just to make those two lines like list items.